a/unit-tests/lbge-unittests.asd

```diff
@@ -0,0 +1,9 @@
+(defpackage :lbge-unit-tests-system
```

Comment:
After changes from _unittests_ to _unit-tests_ this file still has _unittests_ in its name.

Reply:
Really. Knew I've missed something.